globals_i_test.go

```diff
@@ -23,8 +22,7 @@ func integrationTestsBuildCluster() *Cluster {
 	var cluster *Cluster
 	var err error
 	nodeOpts := &NodeOptions{
-		RemoteAddress:  getRiakAddress(),
-		RequestTimeout: time.Second * 20,
```

Comment:
This was "artificially" set to 20 secs to prevent socket timeouts. No longer necessary as the operation timeout will be used if it's greater than the socket timeout.